streams/src/main/java/org/apache/kafka/streams/kstream/internals/WindowedSerializer.java

```diff
@@ -17,13 +17,19 @@
 
 package org.apache.kafka.streams.kstream.internals;
 
+import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.config.ConfigException;
 import org.apache.kafka.common.serialization.Serializer;
-import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.common.utils.Utils;
 import org.apache.kafka.streams.kstream.Windowed;
 
 import java.nio.ByteBuffer;
 import java.util.Map;
 
+/**
+ *  The inner serializer class must be specified by setting the property key.serializer.inner.class,
```

Comment:
I did not encounter any checksytle failures. Could you kindly point out which files failed to pass this check?